### ISSUE TITLE
Training on RGB to RGB image sequence translation

### ISSUE THREAD

**Issue by orthodoxality** [fast-vid2vid/fast-vid2vid#8]
Hello.

This looks like a fantastic project, and I love the low latency of operation!

I was wondering whether there was a method for training these networks on direct image-sequence to image-sequence translation?  pix2pixHD allows for the use of "--label_nc 0 --no_instance", which facilitate translation between train_A and train_B.

Is there an equivalent which would allow me to harness the power of your project?

Thank you.

**Comment by fast-vid2vid** [comment]
hi @orthodoxality , actually, you can try RGB-to-RGB for training and testing, since in Pose2Body application. We also use the RGB2RGB setting. For a fair comparison, we use semantic maps which are the same as the original setting of Vid2Vid. In the practical application, RGB2RGB may be acceptable.

**Comment by orthodoxality** [comment]
Amazing!  I've placed my inputs in my_dataset/train_img/test/, and my outputs in my_dataset/train_densepose/test and my_dataset/train_openpose/test/.

I have tested a few environments, and am using an 24GB RTX 2080 with a fresh install of 18.04 with CUDA 10.2 installed.  Anaconda was unable to provide the exact requirements but I was able to get a Pytorch 1.12 environment to install flownet2_pytorch and run train.py, which seems to almost begin the training process with --dataroot, --name, and --label_nc 0.

Unfortunately, I am receiving the same fp16 error as kjgfxzp, which doesn't seem to respond to input flags.  I have installed apex in other environments, but I gather from kjgfxzp's issue that it has been removed and shouldn't be used.

May I ask how this was resolved?  Or perhaps more information about successful training environment or suitable docker container that has been used?

Thank you again.  Thank you so much for your work, patience, and help.
